cmd/compose/create.go

```diff
@@ -110,8 +110,10 @@ func (opts createOptions) GetTimeout() *time.Duration {
 func (opts createOptions) Apply(project *types.Project) {
 	if opts.Pull != "" {
```

Comment:
I guess this whole loop can be skipped if `opts.PullChanged` isn't set?

(that said; is there code elsewhere that sets the default if it's not defined in the service)?

Reply:
Right, it's too late, I should go to bed 😅